### PR TITLE
Modified intensity_black_body and calculate_luminosity to use Numba

### PR DESCRIPTION
**Pull request by BugsBunny0**

### CODE REVIEW

File: tardis/util/base.py [hunk 1]
```diff
@@ -121,7 +122,7 @@ def roman_to_int(roman_string):
         raise ValueError('Can not interpret Roman Numeral {0}'.format(roman_string))
     return result
 
-
+@njit(error_model = 'numpy')
```
Comment (wkerzendorf) [inline tardis/util/base.py]: why do you use this?

File: tardis/util/base.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 import re
 from collections import OrderedDict
 
-import numexpr as ne
+# import numexpr as ne
```
Comment (wkerzendorf) [inline tardis/util/base.py]: good!

File: tardis/util/base.py [hunk 3]
```diff
@@ -242,7 +243,7 @@ def create_synpp_yaml(radial1d_mdl, fname, shell_no=0, lines_db=None):
     with open(fname, 'w') as f:
         yaml.dump(yaml_reference, stream=f, explicit_start=True)
 
-
+@njit( error_model = 'numpy')
```
Comment (wkerzendorf) [inline tardis/util/base.py]: please profile the code - does this make it faster?

File: tardis/util/base.py (hunk 2 above)
Comment (BugsBunny0) [reply tardis/util/base.py]: Thanks

**Comment by marxwillia** [comment]
Once you have profiled the code, try to find other functions to optimize.

**Pull request closed without merging**